.devcontainer/website/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Node.js & TypeScript",
+  "image": "mcr.microsoft.com/devcontainers/typescript-node:16-bullseye",
+  "workspaceMount": "source=${localWorkspaceFolder},target=/workspaces/onefetch,type=bind,consistency=cached",
+  "workspaceFolder": "/workspaces/onefetch/docs/vercel",
+  "extensions": ["esbenp.prettier-vscode", "svelte.svelte-vscode"],
+  "postCreateCommand": "npm ci",
+  "postAttachCommand": "npm run dev -- --port 3000",
+  "settings": {
+    "editor.formatOnSave": true
+  },
+  "portsAttributes": {
+    "3000": {
+      "label": "onfetch.dev",
+      "onAutoForward": "notify"
+    }
+  }
```

Comment:
Why not keep the default port 5173?

Reply:
I've been surprised before by changes to the default port, so this is just to keep things a bit more consistent.